model/span.go

```diff
@@ -144,34 +160,34 @@ func (s *Span) ReplaceParentID(newParentID SpanID) {
 }
 
 // GetSamplerParams returns the sampler.type and sampler.param value if they are valid.
-func (s *Span) GetSamplerParams(logger *zap.Logger) (string, float64) {
+func (s *Span) GetSamplerParams(logger *zap.Logger) (SamplerType, float64) {
 	tag, ok := KeyValues(s.Tags).FindByKey(jaeger.SamplerTypeTagKey)
 	if !ok {
-		return "", 0
+		return 0, 0
```

Comment:
```suggestion
		return SamplerTypeUnrecognized, 0
```
this is why we're defining named constants